### ISSUE TITLE
Accessibility: reporting events

### ISSUE THREAD

**Issue by Noimot** [jupyterhub/outreachy#105]
### What page is this for?

https://jupyter.org/governance/conduct/reporting_events.html

### WAVE accessibility report

https://res.cloudinary.com/noimot/image/upload/v1666344666/Screenshot_77_y4anmg.png
![Screenshot (77)](https://user-images.githubusercontent.com/36402553/197164843-e57cfcc9-8eea-4567-a5cc-45a8d941e82d.png)

A form label is present but does not contain any content.

A button is empty or has no value text.
![Screenshot (79)](https://user-images.githubusercontent.com/36402553/197171874-13d6fdd3-9356-4b4c-a843-303b972b2285.png)

### The HTML element to be changed

```html
<label for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</label>

<html></html>

<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>
```

### Your proposed HTML (or CSS), after change

```html
A descriptive should be used instead of an icon or better still an aria-label attribute could be placed inside the input element.

Identify the document language using the <html lang> attribute with a valid value (e.g., <html lang="en">). Ensure that all lang attribute values are valid.

Place text content within the <button> element or give the <input> element a value attribute. or use a div and add role=" button" to the div
```

**Comment by minrk** [comment]
Thank you for opening this issue! We asked for _specific_ HTML changes to be made. Can you provide the actual _fixed HTML_ that you propose?

**Comment by Noimot** [comment]
on the issue raised about <button> being empty because it has no text, we can use a <div> with an attribute role="button"

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
